roles/fio/tasks/build.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# (c) Copyright IBM Corporation
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Copy Fio Container files
+  become: true
+  ansible.builtin.copy:
+    src: files
+    dest: "{{ neerali_fio_artifactdir }}/files"
+    owner: root
+    group: root
```

Comment:
owner and group would need to be the ansible_user_id or ansible_user_gid

Reply:
Done